tutorial.proto

```diff
@@ -14,13 +14,39 @@
 
 syntax = "proto3";
 
-package devrel.tutorial;
+package devrel.codelab;
+
+// TODO: reverse listing order to make reading this more intuitive
+
+message Fragment {
+  // TODO
+}
```

Comment:
If this has no definition and nothing is using it then we don't really need it. If we update this so that something is referencing it then we can keep it, but add a comment stating what it will become (right now I don't know what it is).